src/js/events.js

```diff
@@ -317,11 +375,37 @@
         },
 
         attachToEachElement: function (name, handler) {
+            // build our internal cache to know which element got already what handler attached
+            if (!this.eventsCache) {
+                this.eventsCache = {};
+            }
+
             this.base.elements.forEach(function (element) {
+                var existingCache = this.eventsCache[element.getAttribute('medium-editor-index')];
```

Comment:
Another way to approach making sure that all events handlers are attached to added elements is to modify this method to just cache all the handlers that had been passed to it in the past (maybe just an array of handler methods).

Then, when a new element is passed, we can just loop through the cached handlers and attach each handler to the newly added element.